### DISCUSSION TITLE
pymagnum on ubuntu 23.04 does not work anymore.

### DISCUSSION THREAD

**Discussion by ve2mxo** [CharlesGodwin/pymagnum#51] [General]
After upgrading ubuntu to 23.04, the magnum code has been flush and impossible to install again.

**Comment by CharlesGodwin** [comment]
I'm not sure what would cause the problem. I don't have 23.04 installed to test but try this.

- Remove pymagnum `sudo pip3 uninstall pymagnum`. This may require a response to a delete question. If pymagnum was flushed you may get an error message that you can ignore.
- Reinstall pypmagnum. There are two versions. 1.1.3 that was the default until June 5, 2022.. The current version is 2.0.2.
Install the older version using `sudo pip3 install pymagnum==1.1.3` or the current version with `sudo pip3 install pymagnum`.

If you get an error let me know but please include the command you used to install and all the error messages you got trying to install.
